docs/docs/api_docs/dataframe_lib/dataframe/apply.md

```diff
@@ -33,6 +42,9 @@ See [`pandas.DataFrame.apply`](https://pandas.pydata.org/docs/reference/api/pand
 
 : __args : *tuple*:__ Additional positional arguments to pass to *func*.
 
+: __engine : *{'bodo', 'python', 'numba'}, default 'bodo'*:__ The engine to use to compute the UDF. By default, `engine="bodo"` will apply bodo.jit
+to `func` with fallback to python described above. Use engine='python' to avoid any jit compilation. `engine='numba'` will trigger a fall back to [`pandas.DataFrame.apply`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html#pandas.DataFrame.apply).
```

Comment:
```suggestion
to `func` with fallback to Python described above. Use engine='python' to avoid any jit compilation. `engine='numba'` will trigger a fall back to [`pandas.DataFrame.apply`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html#pandas.DataFrame.apply).
```